src/routes/houses.routes.js

```diff
@@ -22,6 +23,16 @@ const createNewHouseRoutes = (router) => {
 
     return res.status(StatusCodes.OK).json(response);
   });
+
+  router.put('/create-new-house/:id', async (req, res) => {
```

Comment:
I think that the method should be `patch` because you are updating an object and not replacing it with a new one.

Reply:
What about `auth`? You could then get id of the logged user from `req.user._id`